samigo/samigo-app/src/webapp/css/tool_sam.css

```diff
@@ -537,7 +537,7 @@ a.hideDivision {
 	display:inline;
 	float:unset;
 }
-.mcAnswerText label {
+.mcAnswerText label, .mcAnswerText .samigo-answer-label, .mcAnswerText .samigo-answer-text, .tableTF label {
 	font-weight: normal;
 }
```

Comment:
this seems to be moving logic out of SASS back into CSS! We prefer going the other direction.

Reply:
There isn't any SASS file on SAMIGO...  I don't quite understand exactly what you want to fix.
Any help?

Reply:
`/library/src/morpheus-master/sass/modules/tool/samigo/_samigo.scss`

Reply:
Thanks @bjones86. Solved!